components/fxa-client/ios/FxAClient/RustFxAFFI.h

```diff
@@ -57,8 +57,11 @@ FxARustBuffer fxa_get_access_token(FirefoxAccountHandle handle,
                                    const char *_Nonnull scope,
                                    FxAError *_Nonnull out);
 
-FxARustBuffer fxa_clear_access_token_cache(FirefoxAccountHandle handle,
-                                             FxAError *_Nonnull out);
```

Comment:
Does this change need to be called out, or is it hidden behind the Swift wrapper and hence invisible to consumers?

Reply:
This was a mistake actually, `fxa_clear_access_token_cache` never returned a protobuf.